mlflow/pyfunc/backend.py

```diff
@@ -64,11 +66,13 @@ def serve(self, model_uri, port, host):
         # NB: Absolute windows paths do not work with mlflow apis, use file uri to ensure
         # platform compatibility.
         local_uri = path_to_local_file_uri(local_path)
-        command = ("gunicorn --timeout 60 -b {host}:{port} -w {nworkers} "
+
+        command = ("gunicorn --timeout 60 -b {host}:{port} -w {nworkers} ${{GUNICORN_CMD_ARGS}} "
                    "mlflow.pyfunc.scoring_server.wsgi:app").format(
             host=host,
             port=port,
-            nworkers=self._nworkers)
+            nworkers=self._nworkers,
+            gunicorn_cmd_args=gunicorn_cmd_args)
```

Comment:
The `gunicorn_cmd_args` Python variable doesn't exist, right? I think we should remove this format argument.

Reply:
I added it back in. ${{GUNICORN_CMD_ARGS}} does not get interpreted properly by the shell. I had to extract it using os.environ.get("GUNICORN_CMD_ARGS", ""), then append it using .format(). Also there cannot be more than one whitespace between arguments, so I had to go back and make another change.